template/.github/workflows/nightly_at_main.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - uses: actions/checkout@v4
       - name: Get Python version for other CI jobs
         id: vars
-        run: echo "min_python=$(cat .github/workflows/python-version-ci)" >> $GITHUB_OUTPUT
+        run: echo "min_python=$(< .github/workflows/python-version-ci)" >> "$GITHUB_OUTPUT"
```

Comment:
Double quotation marks were added to prevent globbing and splitting. (https://www.shellcheck.net/wiki/SC2086)